doc/rst/developer/chips/README.md

```diff
@@ -20,3 +20,4 @@ See the first CHIP for an overview.
 * [16 Optimizing Record and Array Copies in Chapel](16.rst)
 * [17 GPU Programming Features](17.rst)
 * [18 NUMA-Aware Array Storage](18.rst)
+
```

Comment:
running `make update` generated this newline.. I suppose the last update was done by hand..